.goreleaser.yml

```diff
@@ -1,76 +1,95 @@
 # GoReleaser config
 
+---
 before:
   hooks:
     - go mod tidy
 
 builds:
-- id: pop_darwin
-  binary: soda
-  main: soda/main.go
-  goos:
-    - darwin
-  goarch:
-    - amd64
-  env:
-    - CGO_ENABLED=1
-    - CC=o64-clang
-    - CXX=o64-clang++
-  flags:
-    - -tags
-    - sqlite
+  - id: pop_darwin
+    binary: soda
```

Comment:
I changed the indentation as my vimconfig make it obnoxious.  Maybe I should undo that because this looks more gnarly than it is.